doc/make.bat

```diff
@@ -0,0 +1,49 @@
+@ECHO OFF
```

Comment:
Is this file based off the make.bat that sphinx-quickstart generates or is it something you cooked up yourself?

Reply:
I started with make.bat made by sphinx-quickstart, then added parameters like livehtml and clean